packages/cardhost/app/templates/components/card-schema-updator.hbs

```diff
@@ -4,42 +4,103 @@
 
 <br>
 
-<CardRenderer
-  @card={{this.card}}
-  @format="isolated"
-  @mode="schema"
-  @removeField={{action this.removeField}}
-  @setFieldName={{action this.setFieldName}}
-  @setNeededWhenEmbedded={{action this.setNeededWhenEmbedded}}
-  @setPosition={{action this.setPosition}}
->
-  <div class="card-schema--field-btn-wrapper">
-    {{#unless this.cardstackSession.isAuthenticated}}
-      (Must be logged in to save the card.)
-    {{/unless}}
-    <span data-test-card-schema-is-dirty={{this.isDirtyStr}}>
-      {{#if this.card.isDirty}}(unsaved){{/if}}
-    </span>
-    <button {{on "click" this.cancel}}
-      data-test-card-schema-canel-btn
-      disabled={{not this.cardstackSession.isAuthenticated}}
-    >
-      Cancel
-    </button>
-    <button {{on "click" this.save}}
-      data-test-card-schema-save-btn
-      disabled={{not this.cardstackSession.isAuthenticated}}
-    >
-      Save Card
-    </button>
+<div class="card-manipulator--container">
+
+  {{!-- Left edge --}}
+  <div class="card-manipulator--left-edge">
+    {{!-- Profile sidebar --}}
+    <div class="card-manipulator--left-edge--profile">
+      <div class="profile icon"></div>
+      <div class="settings icon active"></div>
+      <div class="search icon"></div>
+    </div>
+    {{!-- Catalog fields list --}}
+    <div class="card-manipulator--left-edge--catalog">
+      <div class="title">Card Catalog</div>
+
+      {{#each this.fieldComponents as |field|}}
+        <div
+          class="card-manipulator--left-edge--catalog-field"
+          role="button"
+          draggable="true"
+          onmousedown={{action this.beginDragging field}}
+          data-test-card-add-field-draggable={{field.type}}
```

Comment:
As I was mentioning in the `card-manipulator.js`, let's use the new helpers and modifiers for octane to add a draggend event handler that is closed over the `CardManipulatorComponent` so `this` resolves to the `CardManipulator` in the action handler. 

```hbs
{{on "dragend" (action this.endDragging)}}
```
(where `CardManipulatorComponent.isDragging()` is a function that takes no params and does the thing you want to do when dragging ends)

This will invoke `CardManipulatorComponent.endDragging()` with the `this` of `endDragging()` bound to the `CardManipulatorComponent`. This prevents the need to add event listeners to the `window`, as well as worry about removing event listeners--as you should always pair an `addEventListener()` with a `removeEventListener()` otherwise you'll have event listener functions firing multiple times for each time you have invoked `addEventListener()`.

In this case I think you just want to set `this.isDragging` to `false` when dragging stops. For which a `mut` will work perfectly fine. So likely this is even more succinct and requires no javascript at all:

```hbs
{{on "dragend" (action (mut this.isDragging) false)}}
```

Here's the reference for `{{on}}`: https://api.emberjs.com/ember/3.11/classes/Ember.Templates.helpers/methods/on?anchor=on

Here's the octane reference for `{{action}}` https://octane-guides-preview.emberjs.com/release/templates/actions/.  `{{action}}` in octane has been _greatly_ simplified so that it's only job is to do the template equivalent of this `() => { ... }`. It doesn't do anything else.

Here's the reference for `{{fn}}` which you can use to pass arbitrary things into the function like perhaps the `{{field}}` that is being emitted from the `{{#each}}`: https://api.emberjs.com/ember/3.11/classes/Ember.Templates.helpers/methods/on?anchor=fn The job of this guy is to pass additional things into your function (currying). So if you want to pass more than just the event, you can use this to add the additional items to pass into the function. Additionally, `{{fn}}` will close over the function so that `this` inside your function will refer to the `CardManipulatorComponent`. You can see an example of this for a bunch of the other `CardManipulatorComponent` actions:  https://github.com/cardstack/cardstack/blob/master/packages/cardhost/app/templates/components/field-renderer.hbs#L76 https://github.com/cardstack/cardstack/blob/master/packages/cardhost/app/templates/components/field-renderer.hbs#L91